variables.tf

```diff
@@ -0,0 +1,51 @@
+variable "automation_account_name" {
+  type        = string
+  description = "name of automation account"
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "name of resource group"
+}
+
+variable "location" {
+  type        = string
+  description = "azure location"
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "tags"
+}
+
+variable "automation_credentials" {
```

Comment:
could this be a list(object) so we know what the parameters should be e.g. name, username, password etc with validation?

Reply:
done